academic-services/graph/samples-azure-data-lake-hindex.md

```diff
@@ -0,0 +1,347 @@
+       SELECT
+           AuthorId,
+           PaperId,
+           EstimatedCitation,
+           ROW_NUMBER() OVER(PARTITION BY AuthorId ORDER BY EstimatedCitation DESC) AS Rank
+       FROM @AuthorPaperCitation;
+   
+   // Generate author hindex
+   @AuthorHIndexTemp =
+       SELECT
+           AuthorId,
+           SUM(EstimatedCitation) AS TotalEstimatedCitation,
+           MAX(CASE WHEN EstimatedCitation >= Rank THEN Rank ELSE 0 END) AS HIndex
+       FROM @AuthorPaperOrderByCitation 
+       GROUP BY AuthorId;
+   
+   // Get author detail information
+   @AuthorHIndex =
+       SELECT
+           I.AuthorId,
+           A.DisplayName,
+           F.DisplayName AS AffiliationDisplayName,
+           A.PaperCount,
+           I.TotalEstimatedCitation,
+           I.HIndex
+       FROM @AuthorHIndexTemp AS I
+       INNER JOIN @Authors AS A
+           ON A.AuthorId == I.AuthorId
+       LEFT OUTER JOIN @Affiliations AS F
+           ON A.LastKnownAffiliationId == F.AffiliationId;
+   
+   OUTPUT @AuthorHIndex
+   TO @OutAuthorHindex
+   ORDER BY HIndex DESC, AuthorId
+   FETCH 100 ROWS
+   USING Outputters.Tsv(quoting : false);
+   ```
+
+1. In this code block, replace `<AzureStorageAccount>`, and `<MagContainer>` placeholder values with the values that you collected while completing the prerequisites of this sample.
+
+   |Value  |Description  |
+   |---------|---------|
+   |**`<AzureStorageAccount>`** | The name of your Azure Storage (AS) account containing MAG dataset. |
+   |**`<MagContainer>`** | The container name in Azure Storage (AS) account containing MAG dataset, Usually in the form of **mag-yyyy-mm-dd**. |
+
+1. Provide a **Job name**, change **AUs** to 5, and select **Submit**.
+
+   ![Submit AuthorHIndex job](media/samples-azure-data-lake-hindex/author-hindex-submit.png "Submit AuthorHIndex job")
+
+1. The job should finish successfully in about 10 minutes.
+
+   ![AuthorHIndex job status](media/samples-azure-data-lake-hindex/author-hindex-status.png "AuthorHIndex job status")
+
+## View output data
+
+The output of the ADLA job in previous section goes to "/Output/AuthorHIndex.tsv" in the Azure Data Lake Storage (ADLS). In this section, you use [Azure portal](http://portal.azure.com/) to view output content.
+
+1. In the [Azure portal](https://portal.azure.com), go to the Azure Data Lake Storage (ADLS) service that you created, and select **Data Explorer**.
+
+   ![Data Explorer](media/samples-azure-data-lake-hindex/adls-data-explorer.png "Data Explorer")
+
+1. Select **Output** > **AuthorHIndex.tsv**.
+
+   ![AuthorHIndex.tsv](media/samples-azure-data-lake-hindex/adls-data-explorer-2.png "AuthorHIndex.tsv")
+
+1. You see an output similar to the following snippet:
+
+   ![AuthorHIndex.tsv preview](media/samples-azure-data-lake-hindex/adls-file-preview.png "AuthorHIndex.tsv preview")
+
+## Next steps
+
+Advance to the next step to learn about analytics and visualization using Azure Data Lake Analytics.
```

Comment:
I'd rewrite this section to say:
"If you're interested in Academic analytics and visualization, we have created U-SQL samples that use some of the same functions referenced in this tutorial. "